pkg/app/api/service/webservice/service.proto

```diff
@@ -197,7 +197,8 @@ message UpdateApplicationRequest {
     string name = 2 [(validate.rules).string.min_len = 1];
     string env_id = 3 [(validate.rules).string.min_len = 1];
     string piped_id = 4 [(validate.rules).string.min_len = 1];
-    model.ApplicationGitPath git_path = 5 [(validate.rules).message.required = true];
+    // TODO: Rmove right after removing the web form for Git path
```

Comment:
```suggestion
    // TODO: Remove right after removing the web form for Git path
```

Reply:
Now the web's PR has been merged so we can remove this TODO.

Reply:
I noticed your comment just now. Let me do so right now.